src/containers/Masthead/components/MastheadSearchForm.jsx

```diff
@@ -74,12 +75,21 @@ export class MastheadSearchForm extends Component {
   }
 
   handleSubmit(evt) {
+    const { history, locale } = this.props;
     evt.preventDefault();
     const isNDLAUrl = /^https:\/(.*).ndla.no\/(article|subjects|nb|nn|en)\/(node|\d*)(\/|\d*)/.test(
       this.state.query,
     );
+    const isNodeId =
+      this.state.query.length > 2 &&
```

Comment:
Kanskje assigne this.stat.query til en query variabel, så blir koden litt enklere å lese.